app/build.gradle

```diff
@@ -170,7 +170,7 @@ dependencies {
     implementation "com.github.AntennaPod:AntennaPod-AudioPlayer:$audioPlayerVersion"
     implementation 'com.github.mfietz:fyydlin:v0.5.0'
     implementation 'com.github.ByteHamster:SearchPreference:v2.0.0'
-    implementation 'com.github.skydoves:balloon:1.1.5'
+    implementation 'com.github.skydoves:balloon:1.1.7'
```

Comment:
Please remove the dependency updates. They currently work fine and (unless there is a known bug) I do not want to upgrade, especially because we are already in the beta stage. The last dependency update PR caused me some trouble with new bugs.

Reply:
Oh, alright. I'll leave the OkHttp one though if that's ok with you.

Reply:
What is the reason for keeping this (and the Gradle update)? Does it fix a specific error that you experience? I do not want to upgrade dependencies during the beta phase unless there is a very good reason to do it.

Reply:
@ByteHamster I just feel that absolutely nothing will break from this 🤔 . Android Studio is just Android Studio, and won't really have anything to do with the app itself. The two OkHttp releases since 3.12.10 are single-issue bugfixes.

Reply:
> Android Studio is just Android Studio, and won't really have anything to do with the app itself.

It is related to building and publishing the app, though. If there is no good reason to do so, please do not touch any dependencies directly before a release.

Reply:
Mmm, alright then.